install.sh

```diff
@@ -201,13 +205,47 @@ parse_command_line_arguments() {
         prev_p="$p"
     done
 
+    if [ "$SINGLE_REPO_INSTALL" = "yes" ] && ! check_not_deprecated_single_install; then
+        echo "! Install failed due to deprecated single install" >&2
+        exit 1
+    fi
+
     # Using core.hooksPath implies it applies to all repo's
     if [ "$SINGLE_REPO_INSTALL" = "yes" ] && [ "$USE_CORE_HOOKSPATH" = "yes" ]; then
         echo "! Cannot use --single and --use-core-hookspath together" >&2
         exit 1
     fi
 }
 
+############################################################
+# Check if this repo is not a deprecated single install.
+#
+# Returns: 1 if deprecated single install, 0 otherwise
+############################################################
+check_not_deprecated_single_install() {
+    if is_git_repo "$(pwd)" && git config --local githooks.single.install; then
```

Comment:
As mentioned on the test below, we should fail single installs everywhere, not only in already single install repos.
@gabyx can you please change the logic like that?